src/relay/WalletSDKRelay.ts

```diff
@@ -122,6 +125,24 @@ export class WalletSDKRelay extends WalletSDKRelayAbstract {
       this.eventListener,
     );
 
+    this.subscriptions.add(
+      this.connection.sessionConfig$.subscribe({
+        next: sessionConfig => {
+          this.storage.setItem(
+            SESSION_CONFIG_KEY,
```

Comment:
I don't love the idea of storing this but it seems like we expose quite a bit of info to caller pages today.

This doesn't expand our risk profile given we expose so much.

<img width="1067" alt="Screen Shot 2022-05-11 at 1 27 58 PM" src="https://user-images.githubusercontent.com/3277097/167920483-780242bf-2a8f-414e-8055-114fd8ab3418.png">

Reply:
I wish I understood all this stuff